src/js/popup.js

```diff
@@ -51,11 +51,39 @@ function showSignUpPanel() {
 }
 
 
-function choosePanel(numRemaining, panelId, premium, premiumEnabledString){
-  if (premium && premiumEnabledString === "True"){
+function premiumFeaturesAvailable(premiumEnabledString) {
+  if (premiumEnabledString === "True") {
+    return Boolean(true);
+  }
+  else {
+    return Boolean(false);
+  }
+}
+
+function enablePremium(premium, premiumEnabledString) {
```

Comment:
This is function is to check if 1) user is premium 2) premium features are available. Different from the earlier function which only checks the `premiumEnabledString` string, since I have to use that for a specific case.